BUILD.bazel

```diff
@@ -0,0 +1,289 @@
+        "src/tbb/rml_tbb.h",
+        "src/tbb/rml_thread_monitor.h",
+        "src/tbb/rtm_mutex.cpp",
+        "src/tbb/rtm_rw_mutex.cpp",
+        "src/tbb/scheduler_common.h",
+        "src/tbb/semaphore.cpp",
+        "src/tbb/semaphore.h",
+        "src/tbb/small_object_pool.cpp",
+        "src/tbb/small_object_pool_impl.h",
+        "src/tbb/task.cpp",
+        "src/tbb/task_dispatcher.cpp",
+        "src/tbb/task_dispatcher.h",
+        "src/tbb/task_group_context.cpp",
+        "src/tbb/task_stream.h",
+        "src/tbb/thread_data.h",
+        "src/tbb/tls.h",
+        "src/tbb/version.cpp",
+        "src/tbb/waiters.h",
+    ],
+    hdrs = [
+        "include/tbb/blocked_range.h",
+        "include/tbb/blocked_range2d.h",
+        "include/tbb/blocked_range3d.h",
+        "include/tbb/blocked_rangeNd.h",
+        "include/tbb/cache_aligned_allocator.h",
+        "include/tbb/combinable.h",
+        "include/tbb/concurrent_hash_map.h",
+        "include/tbb/concurrent_lru_cache.h",
+        "include/tbb/concurrent_map.h",
+        "include/tbb/concurrent_priority_queue.h",
+        "include/tbb/concurrent_queue.h",
+        "include/tbb/concurrent_set.h",
+        "include/tbb/concurrent_unordered_map.h",
+        "include/tbb/concurrent_unordered_set.h",
+        "include/tbb/concurrent_vector.h",
+        "include/tbb/enumerable_thread_specific.h",
+        "include/tbb/flow_graph.h",
+        "include/tbb/flow_graph_abstractions.h",
+        "include/tbb/global_control.h",
+        "include/tbb/info.h",
+        "include/tbb/memory_pool.h",
+        "include/tbb/null_mutex.h",
+        "include/tbb/null_rw_mutex.h",
+        "include/tbb/parallel_for.h",
+        "include/tbb/parallel_for_each.h",
+        "include/tbb/parallel_invoke.h",
+        "include/tbb/parallel_pipeline.h",
+        "include/tbb/parallel_reduce.h",
+        "include/tbb/parallel_scan.h",
+        "include/tbb/parallel_sort.h",
+        "include/tbb/partitioner.h",
+        "include/tbb/profiling.h",
+        "include/tbb/queuing_mutex.h",
+        "include/tbb/queuing_rw_mutex.h",
+        "include/tbb/scalable_allocator.h",
+        "include/tbb/spin_mutex.h",
+        "include/tbb/spin_rw_mutex.h",
+        "include/tbb/task.h",
+        "include/tbb/task_arena.h",
+        "include/tbb/task_group.h",
+        "include/tbb/task_scheduler_observer.h",
+        "include/tbb/tbb.h",
+        "include/tbb/tbb_allocator.h",
+        "include/tbb/tbbmalloc_proxy.h",
+        "include/tbb/tick_count.h",
+        "include/tbb/version.h",
+    ],
+    copts = [
+        "-D__TBB_BUILD=1",
+        "-w",
+    ],
+    defines = [
+        "USE_PTHREAD",
+        "__TBB_BUILD=1",
+        "TBB_PREVIEW_GRAPH_NODES=1",
+        "__TBB_NO_IMPLICIT_LINKAGE",
+    ],
+    includes = [
+        "include",
+    ],
+    linkopts =
+        select({
+            ":windows": [],
+            "//conditions:default": [
+                "-ldl",
+                "-lpthread",
```

Comment:
Consider `-pthread` (likely it should be added to `copts` as well). [Here](https://stackoverflow.com/questions/23250863/difference-between-pthread-and-lpthread-while-compiling) is an explanation about difference between `-lpthread` and `-pthread`. CMake configuration uses `find_package(Threads)` that finally inserts `-pthread` to compilation and link lines.

Reply:
Change to `-pthread`